packages/vuelidate/src/core.js

```diff
@@ -449,10 +450,9 @@ export function setValidations ({ validations, state, key, parentKey, childResul
   }) : null
 
   if (config.$autoDirty) {
```

Comment:
Have we tested if this scenario works?

```js
validations: () => ({
  name: { required },
  surname: { anything },
  $autoDirty: true
})
```

Reply:
Good question. I think I have not tested this or I don’t remember. But this is also something I was considering. Or have the 3rd argument in `useVuelidate` by the config object. Wdyt? Having those next to actual rules doesn’t really make that much sense.

Reply:
Nope.. I dont like it either.... But what about options API? Dont treat it as a second class citizen mate :D

Reply:
Good idea, but for another PR.